app/_hub/kong-inc/request-validator/index.md

```diff
@@ -32,14 +31,16 @@ params:
   consumer_id: true
   config:
     - name: body_schema
-      required: true
+      required: false
```

Comment:
`required = false` is not quite correct, see my other comment

Reply:
Changed to semi